Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM funkyfuture/nest-of-serpents
```

Comment:
This stuff is only for running tests, and in an environment that's supposed to be isolated from the rest of your system, so the security risk is very low, probably not even worth a warning.